entrypoint

```diff
@@ -106,7 +106,12 @@ if [[ -n "$DB_RESTORE_TARGET" ]]; then
   if [ -d /scripts.d/pre-restore/ ]; then
     for i in $(ls /scripts.d/pre-restore/*.sh); do
       if [ -x $i ]; then
-        DB_RESTORE_TARGET=${DB_RESTORE_TARGET} DB_DUMP_DEBUG=${DB_DUMP_DEBUG} $i
+        #The script must output the name of the sql dump so processing
+        #continues as usual
+        DB_RESTORE_TARGET=$(DB_RESTORE_TARGET=${DB_RESTORE_TARGET} DB_DUMP_DEBUG=${DB_DUMP_DEBUG} $i)
```

Comment:
What is this for? Why capturing it into `DB_RESTORE_TARGET`?

Reply:
Because DB_RESTORE_TARGET initially points to the tarball that contains both the sql dump and the files tarball, so to get entrypoint working as usual after pre-restore DB_RESTORE_TARGET must have the name of the sql dump file.